templates/static/styles.css

```diff
@@ -1,108 +1,118 @@
 body {
-    font-family: Arial, sans-serif;
-    margin: 20px;
+ font-family: Arial, sans-serif;
+ margin: 20px;
 }
 
 h1 {
-    color: #333;
+ color: #333;
 }
 
 form {
-    margin-bottom: 20px;
+ margin-bottom: 20px;
 }
 
 label {
-    display: block;
-    margin-bottom: 5px;
+ display: block;
+ margin-bottom: 5px;
 }
 
 input[type="text"] {
-    width: 100%;
-    padding: 8px;
-    margin-bottom: 10px;
-    border: 1px solid #ccc;
-    border-radius: 4px;
+ width: 100%;
+ padding: 8px;
+ margin-bottom: 10px;
+ border: 1px solid #ccc;
+ border-radius: 4px;
 }
 
 button {
-    padding: 10px 15px;
-    background-color: #007BFF;
-    color: white;
-    border: none;
-    border-radius: 4px;
-    cursor: pointer;
+ padding: 10px 15px;
+ background-color: #007BFF;
+ color: white;
+ border: none;
+ border-radius: 4px;
+ cursor: pointer;
 }
 
 button:hover {
-    background-color: #0056b3;
+ background-color: #0056b3;
 }
 
 #result {
-    white-space: pre-wrap;
-    background-color: #f4f4f4;
-    padding: 10px;
-    border: 1px solid #ddd;
-    border-radius: 4px;
+ white-space: pre-wrap;
+ background-color: #f4f4f4;
+ padding: 10px;
+ border: 1px solid #ddd;
+ border-radius: 4px;
 }
 
 .image-container {
-    position: relative;
-    display: inline-block;
-    margin: 10px 0;
+ position: relative;
+ display: inline-block;
+ margin: 10px 0;
 }
 
 .image-container img {
-    display: block;
-    max-width: 100%;
+ display: block;
+ max-width: 100%;
 }
 
 .draggable {
-    position: absolute;
-    top: 10px; /* Adjust as needed */
-    left: 10px; /* Adjust as needed */
-    cursor: move;
-    user-select: none;
-    border: 1px solid #ccc;
-    border-radius: 3px;
-    background-color: transparent !important; /* Ensure no background */
-    padding: 0 !important; /* Remove padding */
+ position: absolute;
```

Comment:
The indentation is worse now. Fix it.

Reply:
I apologize for the oversight. I'll fix the indentation properly this time. I'll update the PR shortly. Thank you for your patience!